packages/app-mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -41,35 +35,6 @@
 		android:resizeableActivity="true"
 		android:theme="@style/AppTheme">
 
-		<!-- RN-NOTIFICATION -->
-		<receiver
-			android:name="com.emekalites.react.alarm.notification.AlarmReceiver"
-			android:enabled="true"
-			android:exported="true">
-			<intent-filter>
-				<action android:name="ACTION_DISMISS" />
-				<action android:name="ACTION_SNOOZE" />
-			</intent-filter>
-		</receiver>
-
-		<receiver
-			android:name="com.emekalites.react.alarm.notification.AlarmDismissReceiver"
-			android:enabled="true"
-			android:exported="true" />
-			<receiver
-				android:name="com.emekalites.react.alarm.notification.AlarmBootReceiver"
-				android:directBootAware="true"
-				android:enabled="false"
-				android:exported="true">
-			<intent-filter>
-				<action android:name="android.intent.action.BOOT_COMPLETED" />
-				<action android:name="android.intent.action.QUICKBOOT_POWERON" />
-				<action android:name="com.htc.intent.action.QUICKBOOT_POWERON" />
-				<action android:name="android.intent.action.LOCKED_BOOT_COMPLETED" />
-			</intent-filter>
-		</receiver>
-		<!-- /RN-NOTIFICATION -->
-
 		<!--
```

Comment:
That's great if you can remove all this configuration. Does it mean it's now bundled with the extension?

Reply:
Exactly, the final manifest is produced by merging the main one with all packages.